cmd/kubelet/app/options/options.go

```diff
@@ -416,8 +416,8 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 
 	fs.Int32Var(&c.RegistryPullQPS, "registry-qps", c.RegistryPullQPS, "If > 0, limit registry pull QPS to this value.  If 0, unlimited.")
 	fs.Int32Var(&c.RegistryBurst, "registry-burst", c.RegistryBurst, "Maximum size of a bursty pulls, temporarily allows pulls to burst to this number, while still not exceeding registry-qps. Only used if --registry-qps > 0")
-	fs.Int32Var(&c.EventRecordQPS, "event-qps", c.EventRecordQPS, "QPS to limit event creations. The number must be >= 0. If 0 will use DefaultQPS: 5.")
-	fs.Int32Var(&c.EventBurst, "event-burst", c.EventBurst, "Maximum size of a bursty event records, temporarily allows event records to burst to this number, while still not exceeding event-qps. The number must be >= 0. If 0 will use DefaultBurst: 10.")
+	fs.Int32Var(&c.EventRecordQPS, "event-qps", c.EventRecordQPS, "QPS to limit event creations. The number must be >= 0. If 0 will use DefaultQPS: 50.")
```

Comment:
If the proposed changes are made in `server.go`, then isn't this incorrect?  It should then be:

"Default value is 50. If 0, rate limiting is not enforced." 

Further, if we are mapping 0 => -1 in the `server.go` code, why are we even restricting the input value be >=0 ?

Couldn't we just say, "A value of -1 allows unlimited QPS"?

Reply:
Sounds good, but the flag is deprecated, do we still need to change the description?

@pacoxu